app/Http/Controllers/CampController.php

```diff
@@ -159,6 +159,27 @@ public function store(Request $request, Validate $validate)
                 $message = trans('message.error.invalid_data');
                 return $this->resProvider->apiJsonResponse($status, $message, $result, $error);
             }
+            if(isset($request->parent_camp_num) && !empty($request->parent_camp_num)){
+                $parentCamp = Camp::where('camp_num',$request->parent_camp_num)->where('topic_num',$request->topic_num)->first();
+            }
+            $topic = Topic::where('topic_num',$request->topic_num)->first();
+            $isDisabled = ($parentCamp->is_disabled == 1 || $topic->is_disabled == 1) ? 1 : 0;
+            $isOneLevel = ($parentCamp->is_one_level == 1 || $topic->is_one_level == 1) ? 1 : 0;
+            //dd($isDisabled);
+
+            if($isDisabled == 1){
```

Comment:
We can use `$parentCamp->is_disabled == 1 || $topic->is_disabled == 1` directly here. No need to define a variable above